src/Cardano/Signer.cpp

```diff
@@ -111,15 +117,15 @@ Common::Proto::SigningError Signer::assembleSignatures(std::vector<std::pair<Dat
         const auto publicKey = privateKey.getPublicKey(TWPublicKeyTypeED25519Extended);
         const auto signature = privateKey.sign(txId, TWCurveED25519Extended);
         // public key (first 32 bytes) and signature (64 bytes)
-        signatures.push_back(std::make_pair(subData(publicKey.bytes, 0, 32), signature));
+        signatures.push_back(make_pair(subData(publicKey.bytes, 0, 32), signature));
```

Comment:
This is inefficient / less readable, because `push_back` doesn't construct the pair in place:

https://godbolt.org/z/5rEv4zKoe

Reply:
Changed to emplace_back